### PR TITLE
Feature/kitosudv 2219 task 2325 termination

### PR DESCRIPTION
**Pull request by JacobMalling**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs [hunk 1]
```diff
@@ -69,8 +69,8 @@ private static ContractTerminationTermsResponseDTO MapTerminationTerms(ItContrac
             return new ()
             {
                 NoticePeriodMonths = contract.TerminationDeadline?.MapIdentityNamePairDTO(),
-                NoticeByEndOf = contract.Running?.ToYearSegmentChoice(), 
-                NoticePeriodExtendsCurrent = contract.ByEnding?.ToYearSegmentChoice()
+                NoticeByEndOf = contract.ByEnding?.ToYearSegmentChoice(),
```
Comment (mrjsawdk) [inline Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs:72]: uhadada

**Review by mrjsawdk — APPROVED** [review]
z2go

File: Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs (hunk 1 above)
Comment (JacobMalling) [reply Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs:72]: Ja der var en lille fejl i response mapperen

Comment (mrjsawdk) [reply Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs:72]: det var heldigt, at den lige blev fanget

Comment (mrjsawdk) [reply Presentation.Web/Controllers/API/V2/External/ItContracts/Mapping/ItContractResponseMapper.cs:72]: vi bør nok rename de der model properties på et tidspunkt, men altså det kræver jo altid lidt mere arbejde pga. ui

**Pull request merged**